lib/Doctrine/Cli.php

```diff
@@ -470,7 +470,10 @@ public function run(array $args)
         } catch (Exception $exception) {
             //Do not rethrow exceptions by default
             if ($this->getConfigValue('rethrow_exceptions', false)) {
-                throw new $exception($this->formatExceptionMessage($exception));
```

Comment:
`$exception` is an Exception object at this point, strange that this worked (tests do hit this).

Changed to be a little bit more explicit in how this exception is re-thrown.